### ISSUE TITLE
[Feature Request]: v1 API supports

### ISSUE THREAD

**Issue by soltanoff** [acheong08/ChatGPT#1130]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Hi! 
Thank you very much for your work!
And I have a question: How can I buy you coffee?

I guess I'm not the only one who wants to support you. :) 

### Proposed workflow

Press F to pay respect.

**Comment by 7trail** [comment]
Agreed. You deserve a ton of support for the work you've done on a 20K+ starred repo.

**Comment by hrizal** [comment]
+1

**Comment by tommyxps** [comment]
+1

**Comment by acheong08** [comment]
Money isn't my main problem but whether I can spend it. I have around $9k stuck in USDT but no platforms are willing to let me withdraw it. I also earn a ton from security bounties but all that is held by my parents so I can't spend it.
